e2e/rsmgp_example_test/test_simple/test.yml

```diff
@@ -0,0 +1,27 @@
+query: >
+    CALL libmemgraph_query_module_in_rust_example.test_procedure() YIELD *;
```

Comment:
@jmatak is it possible to execute more than one query here? It would be good to reload the module before executing the test query.

Reply:
Can you elaborate more on what you mean by executing more than one query, why should you do that and not make another test case?

Reply:
Prior to the execution of a query, I would like to reload the tested module (or all of them). That's not useful inside CI, but during local development, I would like to rerun an e2e test but before running the test query, the module has to be reloaded because it had to be recompiled (Memgraph instances is running in the background with the old module code). Maybe we just add, e.g. `setup_query` and `test_query` (rename `query`)?

Reply:
In addition to result values testing, Memgraph is required to figure out is there any memory leakage related to `mgp_memory`, reporting the leakage status is maybe one more feature of the `e2e` harness.

Reply:
We don't necessarily need different kind of testing queries, but rather we can call mg.load_all() before each run. That way we can ensure that we are loading the latest version of query modules.

Reply:
This is fine because I can reload locally manually, and in the CI it's not required.